core/ai_worker.go

```diff
@@ -209,8 +209,8 @@ func (rwm *RemoteAIWorkerManager) Process(ctx context.Context, requestID string,
 	if !ok {
 		ctxNonRoutine := context.Background()
 		err = rwm.hiveClient.CompleteJob(ctxNonRoutine, jobID, &hive.CompleteJobRequest{
-			TokensUsed: 0,
-			Status:     hive.JobStatusCompleted,
+			Payout: 0,
```

Comment:
Payout is still 0 here,

See here for an example but I think you will have to do something like here, but then fetch the price for a pipeline/capability (not exactly sure on the api here)

Reply:
Yea for ai_worker it's still 0. For transcoding I was assuming a fixed price per pixel and calculating with that.